RTCPeerConnection.js

```diff
@@ -235,9 +235,10 @@ export default class RTCPeerConnection extends EventTarget(PEER_CONNECTION_EVENT
   }
 
   _getTrack(streamReactTag, trackId): MediaStreamTrack {
-    const stream
-      = this._remoteStreams.find(
-          stream => stream._reactTag === streamReactTag);
+    let stream = this._remoteStreams.find(stream => stream._reactTag === streamReactTag);
+    if (!stream) {
+      this._localStreams.find(stream => stream._reactTag === streamReactTag);
```

Comment:
`stream = this._localStreams.find(stream => stream._reactTag === streamReactTag);`